theme-settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * theme-settings.php
+ */
+
+/**
+ * Implements hook_form_system_theme_settings_alter().
+ */
+function europa_form_system_theme_settings_alter(&$form, &$form_state) {
+  // Build the form.
+  if (empty($form['europa'])) {
```

Comment:
The thing here is that we use this europa key in the system setting form coming from different sources, let's say, the same approach has been used in other tickets in order to be sure that there is a fieldset no matter what specific hook implementation is called first.